TTHAnalysis/python/plotter/susy-sos/2los_cuts.txt

```diff
@@ -6,21 +6,23 @@ eventFilters	: $DATA{EventFilters} $MC{1}	; Disable=True
 dilep		: nLepGood == 2
 sublepPt	: (fabs(LepGood2_pdgId)==13 && LepGood2_pt > 3.5) || (fabs(LepGood2_pdgId)==11 && LepGood2_pt > 5)
 
-mll		: minMllSFOS > 4 && minMllSFOS < 50
-upsilonVeto	: minMllSFOS < 9 || minMllSFOS > 10.5
+mll		: mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) > 4 && mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) < 50
+upsilonVeto	: mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) < 9 || mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) > 10.5
 dilepPt		: pt_2(LepGood1_pt, LepGood1_phi, LepGood2_pt, LepGood2_phi) > 3 
 
-ISRjet		: Jet1_jetId >= 4
+ISRjet		: JetSel1_Recl_jetId >= 4
 METoverHT	: (MET_pt/(htJet25j-LepGood1_pt-LepGood2_pt))>(2/3) && (MET_pt/(htJet25j-LepGood1_pt-LepGood2_pt)) < 1.4
 minHT		: (htJet25j-LepGood1_pt-LepGood2_pt) > 100
 
 
 ##### Triggers and Binning
 ### Low MET
-met125		: MET_pt > 125 && metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) > 125 && metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) < 200	; Disable=True
+met125		: MET_pt > 125 && metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) > 125 	; Disable=True
 met125_trig		: (year==2016 && HLT_MuMuMET16) || ( (year==2017 || year==2018) && HLT_MuMuMET )	; Disable=True
-### Med MET	
-met200		: metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) > 200 && metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) < 250	; Disable=True
+
+
+####### Med MET	
+met200		: metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) > 200 	; Disable=True
 met200_col	: metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) > 200 && metmm_pt(LepGood1_pdgId, LepGood1_pt, LepGood1_phi, LepGood2_pdgId, LepGood2_pt, LepGood2_phi, MET_pt, MET_phi) < 300	; Disable=True
```

Comment:
Having only the "col" regions have an upper bound in MET, while the logic for setting the upper bound in MET for all the other regions follow another logic (which is implemented in the sos_plots.py file), seems to me a bit confusing. Is there any way to have the same logic for all the regions?